back/app/models/group.rb

```diff
@@ -75,6 +75,8 @@ def manual?
   end
 
   def update_memberships_count!
+    return if frozen?
```

Comment:
Sometimes, when testing what happens if LogActivityJob runs after record is deleted, I got the following error:
```Ruby
Failure/Error: update!(memberships_count: memberships.where(user: User.registered).count)
     
     FrozenError:
       can't modify frozen attributes
     # ./app/models/group.rb:78:in `update_memberships_count!'
     # ./engines/commercial/smart_groups/app/models/smart_groups/patches/group.rb:80:in `update_memberships_count!'
```